### DISCUSSION TITLE
Is there a guide to upgrade react-native and react-native-windows version?

### DISCUSSION THREAD

**Discussion by FahadMohammedFiroz** [microsoft/react-native-windows#10437] [Q&A] [answered]
I am trying to update my app to use version 0.66 (from 0.63) of react-native-windows, but facing several errors and dependency failures.

**Comment by chrisglein** [comment, marked as answer]
Yes! Check out this article: https://microsoft.github.io/react-native-windows/docs/upgrade-app

**Reply by FahadMohammedFiroz** [reply]
Hello @chrisglein, I was looking for a guide that tracks and/or lists solutions of run-time errors that we face after doing the upgrade.
